### PR TITLE
Fix: Avoid unnecessary imports and aliases

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] proposes an alternative to importing every single class and using unnecessary aliases

💁‍♂️ Also see https://localheinz.com/blog/2020/05/19/avoiding-imports-and-aliases/

### CODE REVIEW

**Comment by jkufner** [comment]
This really reduces readability. There is nothing wrong with a few more imports.

**Comment by javiereguiluz** [comment]
@localheinz thanks for this contribution. @jkufner I agree with you ... but for others this will improve readability, so it's debatable.

However, in this repository we follow Symfony practices as close as possible ... and that includes source code formatting. Given that this differs a lot from the standard Symfony code syntax, I'm afraid we must close without merging. Thanks for understanding it!

**Comment by localheinz** [comment]
@javiereguiluz 

Everything is debatable - even Symfony best practices!

😉

**Comment by javiereguiluz** [comment]
Sure! That's what I tried to say in my comment. We can change Symfony practices ... but we can't do that from this repository, which only follows what Symfony does. Thanks!

File: src/Entity/Post.php [hunk 1]
```diff
@@ -48,86 +47,86 @@ class Post
      *
      * @ORM\Id
      * @ORM\GeneratedValue
```
Comment (jkufner) [inline src/Entity/Post.php:49]: I think you are missing `Mapping` here (and other places): `@ORM\Mapping\GeneratedValue`.

**Comment by jkufner** [comment]
What I really don't like on this style is the use of multi-level namespaces in the code, e.g., `@Validator\Constraints\NotBlank`. It should be `@NotBlank` or `@Constraint\NotBlank`, i.e., use the class directly, or use an alias for the namespace.

On the other hand, I like that there are no aliases, everything is 1:1 with actual namespace names. It may be a good idea to place all annotations into an extra namespace to introduce this alias globally, for example, instead of using `use Symfony\Component\Validator\Constraints as Assert` the annotations could be placed into the `Assert` namespace — `use Symfony\Component\Validator\Constraints\Assert`. Then the annotation would look like `@Assert\NotBlank` without the need to use a custom alias everywhere.

File: src/Entity/Post.php (hunk 1 above)
Comment (localheinz) [reply src/Entity/Post.php:49]: I do - doesn't make any sense to waste my time here, since the pull request has been closed already.

**Pull request closed without merging**